proto/py/compute_image_tools_proto/inspect_pb2.py

```diff
@@ -0,0 +1,382 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: inspect.proto
+"""Generated protocol buffer code."""
+from google.protobuf.internal import enum_type_wrapper
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import message as _message
+from google.protobuf import reflection as _reflection
+from google.protobuf import symbol_database as _symbol_database
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+
+
```

Comment:
All these blank spaces were autogenerated?

Reply:
yeah -- `updatepb.sh` adds a footer to disable flake8, but otherwise leaves the generated code as-is.